### PR TITLE
DEVX-2455: update bootstrapping App Builder app docs with info about template installation

### PR DESCRIPTION
**Pull request by AmyJZhao**
## Description

[DEVX-2455](https://jira.corp.adobe.com/browse/DEVX-2455)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/pages/getting_started/first_app.md [hunk 1]
```diff
@@ -149,8 +149,21 @@ There are two sample flows listed below. Some developers may not have access to
...
+    ```
+    $ aio templates discover
+    ```
+
+9. Install the desired template with the following command:
+
+    ```
+    $ aio templates install <npm package name>
+    ```
+    The template's npm package will be downloaded and extracted, and its installation configuration file parsed for workspace setup, APIs, event registration, and runtime required. `npm install` will run in the background to install the npm package.
```
Comment (slitviachenko) [inline src/pages/getting_started/first_app.md:165]: @AmyJZhao , [@adobe/aio-lib-console-project-installation](https://www.npmjs.com/package/@adobe/aio-lib-console-project-installation) doesn't parse the `event` section, only `workspaces`, `apis` and `runtime`, see https://github.com/adobe/aio-lib-console-project-installation

**Review by slitviachenko — APPROVED** [review]
LGTM 👍

File: src/pages/getting_started/first_app.md [hunk 2]
```diff
@@ -139,29 +143,44 @@ There are two sample flows listed below. Some developers may not have access to
...
+1. Optionally, you can install an App Builder template using Template Registry. Discover available templates in the App Builder template registry with the following command:
+
+    ```bash
+    aio templates discover
+    ```
+
+1. Install the desired template with the following command:
+
+    ```bash
+    aio templates install <npm package name>
```
Comment (shazron) [inline src/pages/getting_started/first_app.md:166]: maybe add `aio templates discover --interactive` here as well?

File: src/pages/getting_started/first_app.md [hunk 3]
```diff
@@ -139,29 +143,44 @@ There are two sample flows listed below. Some developers may not have access to
...
+
+    The template's npm package will be downloaded and extracted, and Developer Console resources such as services and workspaces will be created and configured based on the template's **install.yaml** configuration file. `npm install` will run in the background to install the npm package.
+
 #### 4.1.2 Initializing an empty project
+
 1. In your Terminal, navigate to where you want to initialize your project and type in the following command in your Terminal:
 
-    ```
+    ```bash
     aio app init <app_name> --no-extensions
```
Comment (shazron) [inline src/pages/getting_started/first_app.md:176]: with the cli with Template Registry API integration, `--no-extensions` does not make sense anymore and it has been changed to `--standalone-app`

**Review by shazron — APPROVED** [review]

**Pull request merged**